Add extra info from GitHub using Tags

hhaarsma opened the pull request:
Languages, Teams, DotNet versions

hhaarsma committed:
Add extra info from GitHub using Tags

Languages, Teams, DotNet versions

cliftonc commented on src/github.js:
```diff
@@ -48,6 +48,84 @@ const getRepos = async () => {
+ 
+  const getDotNetVersions = async (repo, keyword) => {
+    try {
+      const { data } = await octokit.request('GET https://api.github.com/search/code?q={keyword}+repo:{repo}', {
```
Is there a reason we need to search vs know where this file is always located?

hhaarsma replied:
Looking into that part (therefor I set the PR as Draft) - when Googling how to get all the csproj files in a GitHub repo the first answers are with search, but I think I have a way of using the Tree API and then get the content for the csproj files.

hhaarsma committed:
Improve C# version detection + fix issue Links

C# version is now done via csproj file content instead of searching

Links seemed to be overlooked due to capitalisation of the name
